bagit.py

```diff
@@ -501,7 +501,7 @@ def files_to_be_fetched(self):
     def has_oxum(self):
         return 'Payload-Oxum' in self.info
 
-    def validate(self, processes=1, fast=False):
+    def validate(self, processes=1, fast=False, completeness=False):
```

Comment:
I'm not entirely in love with just `completness` as the name for this - it's the shortest name but while reading the code I've been thinking it might be better as something like `completeness_only` to make it more obvious that this actually disables checks.